libraries/AP_Motors/AP_MotorsHeli_Dual.cpp

```diff
@@ -592,5 +592,15 @@ bool AP_MotorsHeli_Dual::arming_checks(size_t buflen, char *buffer) const
         return false;
     }
 
+    // Sanity check swash linearierisation angle range
+    if (_swashplate1.get_lin_angle()) {
```

Comment:
```suggestion
    if (_swashplate1.get_lin_angle() > 90.0) {
```

Reply:
I think you should also check for negative numbers, just in case some put in -1 for disabled.

Reply:
fixed with a return early on negative numbers now